src/main/pythonenvselectpopup/pythonenvselectpopup.ts

```diff
@@ -172,6 +182,9 @@ export class PythonEnvironmentSelectPopup {
             <div class="toolbar-button" id="copy-button" onclick='handleCopySessionInfo();' title="Copy session info to clipboard">
               ${copyIconSrc}
             </div>
+            <div class="toolbar-button" id="update-bundled-env-button" onclick='handleUpdateBundledEnv();' title="Update available for Python environment. Click to install.">
```

Comment:
Should it be hidden by default?

```suggestion
            <div class="toolbar-button" id="update-bundled-env-button" onclick='handleUpdateBundledEnv();' style="display: none;" title="Update available for Python environment. Click to install.">
```

It is not clear for me if `onShowUpdateBundledEnvAction` will fire when disabled in options.

Reply:
it is hidden by default already using CSS. you can see in [sessionwindow.ts](https://github.com/jupyterlab/jupyterlab-desktop/pull/764/files#diff-17b1a4383e0d332e3de36bff535ba7f3afb757e7dee8839882b3484c4aea4790) that it is fired only if enabled in options.